charts/inception/values.yaml

```diff
@@ -0,0 +1,495 @@
+# Default values for inception.
```

Comment:
Is there tooling to somehow automatically generate/update this file based on e.g. some Spring metadata?